doc/api/perf_hooks.md

```diff
@@ -95,6 +103,9 @@ added: v8.5.0
 
 * {PerformanceNodeTiming}
 
+> Note: This property is an extension by Node.js. It's not available in Web
+> browsers.
```

Comment:
```suggestion
_This property is an extension by Node.js. It is not available in Web browsers._
```

Explanation: Semantically, `>` is for a block quote. We're not quoting anything here. It's being used to highlight/emphasize. I wouldn't be surprised to find that we do this elsewhere, but I'd prefer we use italics for emphasis, or even nothing at all.